e2e/appActions.js

```diff
@@ -83,7 +83,7 @@ async function createDomainObjectWithDefaults(page, { type, name, parent = 'mine
     ]);
 
     // Wait until the URL is updated
-    await page.waitForNavigation('networkidle');
+    await page.waitForURL(`**/${parentUrl.replace('./', '')}/**?**`);
```

Comment:
This was done because `waitForNavigation` was flaky and we navigate to the base url on each object creation. On navigation to each object URL, if the URL has something following the parent URL, that should indicate that we're hitting the object URL.

Reply:
I'm wondering if we could simplify this even more.

Maybe `**/${parent}/*` ?
Since we know that the parent (whether `mine` or the uuid of another parent) will appear in the new url, following by a `/`, followed by whatever.

Reply:
@ozyx would that be `await page.waitForURL(`**/${parentUrl}/*`);`? That doesn't work for me

Reply:
@khalidadil , well `${parent}` here should be the uuid of the parent. Give `**/${parent}/*` a shot. If that doesn't work, see if `**/${parent}/**` works. The key detail is that we should be waiting for a URL that contains the parent's uuid, followed by a `/`, then followed by anything

Reply:
@khalidadil , Scott has implemented a regex pattern match which you should grep for. It might be better